components/markdown_image.jsx

```diff
@@ -172,6 +175,21 @@ export default class MarkdownImage extends React.PureComponent {
                             }
                         </>
                     );
+
+                    const availableHeight = height ?? imageMetadata.height;
+                    if (availableHeight >= Constants.MAX_INLINE_IMAGE_HEIGHT) {
```

Comment:
Can you expand on the reason to make this conditional? My impression is that embedded images have no such restriction, so it seems weird to be able to collapse some but not others?

If we keep this, I wonder if we can improve on the name `MAX_INLINE_IMAGE_HEIGHT`, which sounds like a bound on how high the inline image can be.

Reply:
I think this is more of a product question. It has been an initial requirement of this ticket to apply this expand/collapse behavior only for images taller than 100px. I assume, there should be some specific use case revolving around using small un-collapsable inline images (maybe some emojis or gifs?). Pinging @esethna for confirmation.

Regarding the name - yes, I got your point. I will work through it, thanks.

Reply:
Fixed.